README.md

```diff
@@ -218,6 +218,38 @@ lambda-cfn save dev
 
 We're happy you want to contribute! Check out [CONTRIBUTING.MD](CONTRIBUTING.MD) for more information.
 
+## Dispatch Integration
+
+You have to set `DispatchSnsArn` variable when deploying your patrol rule and create a message following
+[dispatch message spec](https://github.com/mapbox/dispatch/blob/master/MESSAGE-SPEC.md).
+
+If you want to support both simple patrol rules and dispatch rules you check if `DispatchSnsArn` env variable is set.
```

Comment:
> If you want to support both simple patrol rules and dispatch rules you check if `DispatchSnsArn` env variable is set.

This is really difficult to convey, I'm not sure if this any better, but I'm not sure if anyone knows the difference between `simple patrol rule` and `dispatch rule`.

> Dispatch support can be determined at runtime by checking if the `DispatchSnsArn` environment variable is set. For example: